README.md

```diff
@@ -1,2 +1,18 @@
 # InnerCircle.EmailSender
 Service for sending emails via smtp
+
+## Launch docker containers
+
+1. Change the configuration of credits in the docker-compose file
```

Comment:
Read my comment below about ignoring `docker-compose`.  
I can offer to change a little bit the description:  
1. Create the `docker-compose.yml` file in the source folder
2. Enter credentials of your SendGrid account in `docker-compose.yml`. You have to fill out the following variables: 
  - SendGridOptions__SendGridAPIKey
  - SendGridOptions__SenderEmail
  - SendGridOptions__SenderName